### DISCUSSION TITLE
Not all imported references can be found when downloading Excel

### DISCUSSION THREAD

**Discussion by cdnie** [asreview/asreview#661] [Troubleshooting (report errors in the "Issues" tracker)] [answered]
Another question. :)

My colleague noticed that when downloading the Excel file not all imported references are exported. When downloading the results in a RIS file, Excel only show 897 rows/references from the 10649 imported ones. 

I also noticed when uploading a RIS file I get the notification that it has imported 10649 references. After the Prior Knowledge bit, the graphs show a max of 9764 references. (see jpgs) 

It seems we are losing references along the way. Are we doing something wrong?
Thank you!

![ASreview_RIS after prior knowledge](https://user-images.githubusercontent.com/87352862/125392223-11582e00-e3a6-11eb-9ae8-d8cf4738f630.jpg)
![ASreview_RIS upload](https://user-images.githubusercontent.com/87352862/125392224-11f0c480-e3a6-11eb-8ec9-07047b92c01f.jpg)

**Comment by Rensvandeschoot** [comment]
@cdnie Would you be willing to share your data with me so I can have a look? You can send it to asreview@uu.nl

**Reply by cdnie** [reply]
Of course :) It is a very large file, so hopefully it will arrive. Thank you!!!

**Reply by SagevdBrand** [reply]
Hey there @cdnie, thank you for sharing your file. 
When putting the data into the software, I ran into the same problems you described.
Moreover I noticed the following issues as well:

1. After loading the data into ASReview, some of the statistics are already fully pictured, even though nothing has been reviewed except for two prior inclusions. 

![image](https://user-images.githubusercontent.com/64579032/125954132-ac0b1d53-3985-498b-bb74-808167c912a4.png)

2. After downloading the results in an excel sheet, the column of `included` publications is not present at all. The `asreview_ranking` column is though. 

This leads me to believe that there is some information in the data which causes this odd behavior. 
Within the command line the following warning occurred: 

```
127.0.0.1 - - [2021-07-16 14:32:23] "GET /api/projects HTTP/1.1" 200 3183 0.176043
127.0.0.1 - - [2021-07-16 14:32:47] "POST /api/project/info HTTP/1.1" 201 402 0.003999
127.0.0.1 - - [2021-07-16 14:32:47] "GET /api/project/testing-discussion-issue/info HTTP/1.1" 200 451 0.010004
127.0.0.1 - - [2021-07-16 14:32:47] "GET /static/media/SetUp.93ab4753.svg HTTP/1.1" 200 10820 0.011334
127.0.0.1 - - [2021-07-16 14:32:47] "GET /static/media/roboto-latin-300.ef7c6637.woff2 HTTP/1.1" 200 16104 0.010999
127.0.0.1 - - [2021-07-16 14:32:47] "GET /api/project/testing-discussion-issue/info HTTP/1.1" 200 451 0.003278
127.0.0.1 - - [2021-07-16 14:32:50] "GET /api/project/testing-discussion-issue/info HTTP/1.1" 200 451 0.002999
WARNING:root:Expected start tag in line 0:
 ï»¿TY  - JOUR
INFO:root:Store a copy of the data in a pickle file.
```

I believe therefore that the issue may come from encoding of special signs in the RIS file? @boer0107 These are the first two records. Do you see anything that might cause these problems?

```
TY  - JOUR
AN  - 11010675
CA  - Medline
KW  - Blue Cross Blue Shield Insurance Plans
*Catheterization
Chronic Disease
Clinical Trials as Topic
Device Approval
*Electric Stimulation Therapy
*Hot Temperature/tu [Therapeutic Use]
Humans
Intervertebral Disc
*Low Back Pain/th [Therapy]
Outcome Assessment, Health Care
Technology Assessment, Biomedical
United States
United States Food and Drug Administration
LB  - 11010675
PY  - 2000
SP  - 13-14
ST  - Intradiscal electrothermal therapy for chronic low back pain
T2  - Tecnologica MAP Suppl
TI  - Intradiscal electrothermal therapy for chronic low back pain
UR  - https://ovidsp.ovid.com/ovidweb.cgi?T=JS&CSC=Y&NEWS=N&PAGE=fulltext&D=med4&AN=11010675http://eur.on.worldcat.org/atoztitles/link?sid=OVID:medline&id=pmid:11010675&id=doi:&issn=&isbn=&volume=&issue=&spage=13&pages=13-4&date=2000&title=Tecnologica+MAP+Supplement&atitle=Intradiscal+electrothermal+therapy+for+chronic+low+back+pain.&aulast=&pid=%3Cauthor%3Eanonymous%3C%2Fauthor%3E%3CAN%3E11010675%3C%2FAN%3E%3CDT%3EJournal+Article%3C%2FDT%3E
ID  - 1
ER  - 

TY  - JOUR
AB  - Question: Is a mailed, tailored arthritis self-management programme (SMART) effective, and how does it compare to a conventional arthritis self-management programme (ASMP)? Study design: Two randomised controll ed trials. Main results: SMART versus usual care: at one year, SMART significantly decreased disability, improved role function, reduced depression, and increased self-efficacy compared with usual care (all p < 0.01 except depression, p = 0.04). There were no significant differences between groups for pain, global severity and doctor visits. At two years, SMART decreased global severity (p < 0.01) and number of doctor visits (p < 0.03), and increased self-efficacy (p < 0.001) compared with usual care. There were no significant differences between groups for pain, disability, or depression. At three years, there were no significant differences between groups for any outcome. SMART versus ASMP: at one year, SMART decreased disability (p = 0.02) and increased self-efficacy (p = 0.01) compared with ASMP. Differences in other outcomes were not significant. At two years, there were no significant differences between groups. At three years, ASMP improved role function (p = 0.04) and reduced doctor visits (p = 0.03) compared with SMART. Differences in other outcomes were not significant. Authors' conclusions: SMART is effective and this effect lasts for two-three years without reinforcement compared with usual care. SMART is as effective as ASMP, and could improve accessibility to arthritis self-management treatment. © 2004 Elsevier Ltd. All rights reserved.
CA  - Embase
DB  - Embase
DO  - 10.1016/j.ehbc.2004.08.030
IS  - 5
KW  - article
clinical trial
consultation
controlled study
daily life activity
depression
disability
disease severity
follow up
health care delivery
health program
human
major clinical study
osteoarthritis
pain
patient care
postal mail
priority journal
questionnaire
randomized controlled trial
rheumatoid arthritis
self care
self concept
LA  - English
M3  - Article
N1  - Evidence-Based Healthcare and Public Health
L40298386
2005-03-15
PY  - 2004
SN  - 1744-2249
SP  - 261-262
ST  - Arthritis self-management programme delivered by post is as effective as group teaching
T2  - Evid -Based Healthc Public Health
TI  - Arthritis self-management programme delivered by post is as effective as group teaching
UR  - https://embase.com/search/results?subaction=viewrecord&id=L40298386&from=export
VL  - 8
ID  - 10611
ER  - 
```

**Comment by SagevdBrand** [comment, marked as answer]
With help of @J535D165,  a solution has been found to your issue. As discussed in issue #607, it appears that the information in the LB fields causes some trouble in the software. Only 1's or 0's are expected in this field, instead of 11010675 for example. See #607 for an explanation. Removing the LB information therefore solved the issue and might solve the issue you reported in #660 as well! 

I will send you an email with your data, where the LB columns have been removed.
Hopefully you'll be able to screen flawlessly now!

**Reply by cdnie** [reply]
Thank you so much for your answer!
I use the label field in EndNote where I copy the PMID from the Ascession tag (as one of the deduplication steps in EndNote). I will try to remember to empty that field next time!
Thank you so much everyone @SagevdBrand @J535D165 @Rensvandeschoot for your time in solving this!!

**Reply by SagevdBrand** [reply]
Ah yes, that makes sense. Thank you for posting this issue, this might help other users as well!
